bootstrap.py

```diff
@@ -132,7 +132,7 @@ def update(operation, verbose=None, upgrade=False, offline=False, optional_requi
     # We must install wheel first to eliminate a bunch of scary looking
     # errors at first install.
     # TODO Look towards fixing the packaging so that it works with 0.31
-    pip('install', ['wheel==0.30'], verbose, True, offline=offline)
+    pip('install', ['wheel~=0.32'], verbose, True, offline=offline)
```

Comment:
I ran this on my own Travis account and saw that one of the tests fails because a module from wheels cannot be found. Upgrading to wheel ~0.32 is most likely the cause as is hinted at in the comments section above this line. Moving from 0.30 to 0.32 has caused some [issues as seen in other projects](https://github.com/pypa/wheel/issues/255).

Though out of scope for this PR, packaging will need to be fixed so that tests don't fail for a ModuleNotFound error.  I'll create an issue for this problem.

Reply:
I just read your notes in the message of one of your commits, which stated the reason for upgrading to 0.32.